packages/frontend/src/features/metricsCatalog/components/MetricsCatalogPanel.tsx

```diff
@@ -228,8 +275,24 @@ export const MetricsCatalogPanel = () => {
         } else {
             queryParams.delete('categories');
         }
+
+        if (search) {
+            queryParams.set('search', encodeURIComponent(search));
```

Comment:
we could use this I think next time https://api.reactrouter.com/v7/functions/react_router.useSearchParams.html
`const [searchParams, setSearchParams] = useSearchParams();`